docker/cache/resources/build-cache.sh

```diff
@@ -27,10 +27,11 @@ if [ -z "$(ls -A $GRAPHDB_SOURCE)" ] || [ -n "${FORCE_REFRESH}" ]; then
     ${APP_HOME}/download-data.sh    
 
     #Load all the data into the database (force replace)
-    ${GRAPHDB_HOME}/bin/loadrdf -f -m parallel -c ${REPO_CONFIG} ${GRAPHDB_SOURCE}
+    ${GRAPHDB_HOME}/bin/preload --force -p --chunk 20m -c ${REPO_CONFIG} ${GRAPHDB_SOURCE}
```

Comment:
Never really used preload, but want to experiment with it. Glad to see you have it working.